src/main/java/ua/org/crazy/homework02/Main.java

```diff
@@ -0,0 +1,25 @@
+package ua.org.crazy.homework02;
+
+import static ua.org.crazy.homework02.SortingUtils.*;
```

Comment:
Don't use a wild card in imports. 
https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad